modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/sidebar/Sidebar.es.js

```diff
@@ -82,7 +82,7 @@ export default () => {
 
 								<li className="tbar-item">
 									<ClayButton
-										displayType="unstyled"
+										className="component-action sidebar-close"
```

Comment:
@brunofariasdeo, can you please give me more details about this fix? How did you decide that these classes were the correct ones to be added?

Reply:
@natocesarrego, I believed this close button should have the same behaviour and appearance that form builder sidebar has, hence I've use the same class.